kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/OrderRequestRetriever.java

```diff
@@ -0,0 +1,13 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.time.LocalDate;
```

Comment:
nieuzywany import

Reply:
Jeżeli usunę import java.time.LocalDateTime; to wtedy LocalDateTime orderDate zgłasza błąd:
Error:(9, 9) java: cannot find symbol
  symbol:   class LocalDateTime
  location: class com.kodilla.good.patterns.challenges.challengeTwo.OrderRequestRetriever